using-kubecost/navigating-the-kubecost-ui/savings/pv-right-sizing-rec.md

```diff
@@ -0,0 +1,19 @@
+# Persistent Volume Right-Sizing Recommendations
+
+Kubecost is able to provide recommendations for resizing your PVs by comparing their average usage to their maximum capacity, and can recommend sizing down to smaller storage sizes.
+
+To access the Persistent Volume Right-Sizing Recommendations page, select *Savings* from the left navigation, then select *Right-size persistent volumes*.
+
+![Table](/images/rightsizingpv.png)
+
+Kubecost will display a table containing all PVs in your environment. Table columns include the PV name and its corresponding cluster, and metrics pertaining usage and savings. The estimated savings per month per table item is calculated by subtracting your recommended cost from the current cost.
+
+Recommendations appear in multiple denominations of capacity, rounded up. The smallest denomination Kubecost will recommend per PV is 1.1 GB. From here, the recommended capacity increases in intervals of 1 GB.
```

Comment:
This is more of a front end display issue but I still want to point it out. We are actually calculating disk usage and disk size recommendations in GiB (1024 * 1024 * 1024), but displaying them in the table in GB (1000 * 1000 * 1000) which explains the extra 0.1 GB. While it might appear that we only increment by 1 GB, it would be more accurate to say we increment by 1 GiB.

Reply:
That makes sense, thanks for explaining that. I'll clarify that.